justfile

```diff
@@ -15,6 +15,6 @@ docs-config := base-directory / "docs/mkdocs.yml"
 serve-docs port="8000":
     poetry run mkdocs serve --config-file {{ docs-config }} --dev-addr localhost:{{ port }}
```

Comment:
`mkdocs serve` throws `Error 404` warnings and doesn't display the version dropdown. Can this be fixed?